lib/parser.js

```diff
@@ -197,6 +200,14 @@ module.exports = function parser(identifier, string) {
     })
   }
 
+  // Consumes extend: extend google.protobuf.MessageOptions {optional string type = 50001;}
+  function consumeExtend(parent) {
+    expect(Token.Type.WORD).content
```

Comment:
Wonder if it's worth leaving the unused var here for completeness. `var name = expect(...`